application/classes/Ushahidi/Repository/Config.php

```diff
@@ -49,21 +55,49 @@ public function update(Entity $entity)
 
 		$config = \Kohana::$config->load($group);
 
+		// Intercom count datasources
+		if ($group === 'data-provider') {
+			$intercomData['num_data_sources'] = 0;
```

Comment:
This seems very closely coupled for something that is supposedly loosely coupled...